src/logic/tokens/store/model/token.ts

```diff
@@ -4,7 +4,7 @@ export type TokenProps = {
   address: string
   name: string
   symbol: string
-  decimals: number | string
```

Comment:
If you use this for fetching information from the backend, then be aware that this can be `null`. This happens if the token you try to fetch info for is a `ERC721` token.